BMS_Tx/BMS_test-alerts.cpp

```diff
@@ -0,0 +1,28 @@
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+
+#include "../test/catch.hpp"
+#include "BatteryMonitoringSystem_Main.h"
+
+TEST_CASE("Check if all the values received had been send to console") {
```

Comment:
Neat tests. The crucial functionality here is the output content. Ideally, the output format should be clear by reading the tests. That way, they will document the format and also safeguard it from regressing when future developers modify.